src/utils/upload_conflict_resolution.ts

```diff
@@ -0,0 +1,254 @@
+import {
+	askOnConflicts,
+	errorOnConflict,
+	renameOnConflicts,
+	replaceOnConflicts,
+	ResolveFileNameConflictsParams,
+	ResolveFolderNameConflictsParams,
+	skipOnConflicts,
+	upsertOnConflicts,
+	useExistingFolder
+} from '../types';
+import { NameConflictInfo, FolderNameAndId, FileConflictInfo } from './mapper_functions';
+
+export const resolveFileNameConflicts = async ({
+	wrappedFile,
+	conflictResolution,
+	destinationFileName: destFileName,
+	nameConflictInfo,
+	prompts
+}: ResolveFileNameConflictsParams): Promise<void> => {
```

Comment:
I don't see any references to `this` here. Is this method really needing to be an arrow function?
```suggestion
export async function resolveFileNameConflicts ({
	wrappedFile,
	conflictResolution,
	destinationFileName: destFileName,
	nameConflictInfo,
	prompts
}: ResolveFileNameConflictsParams): Promise<void> {
```

Reply:
SGTM: a5d0d6e